properties/P000135.md

```diff
@@ -2,6 +2,7 @@
 uid: P000135
 name: $R_0$
 aliases:
+  - Symmetric
   - R0
```

Comment:
```suggestion
```

Not relevant to the PR but I think this alias could be removed as unhelpful while we're looking at this.

Reply:
Actually, this is one of those that is not meant as an alias, but rather as an abbreviation.  I.e., we added it for convenience when doing searches in the Explore page, so we only need to type "R0" and not "R_0" and not have to click on a specific property.  There are various others of the same kind: "WLC", "T2", etc.
Do yo think it should be an item of discussion for the next meeting?

Reply:
Thanks for the reminder the rationale here. I'm dismissing this here, but will test to see if the fuzzy matching makes this necessary or not.